File: _1_1_1.java
public class HelloWorld{

     public static void main(String []args){
        if(HelloWorld.checker(args[0]))
            System.out.println("Contains unique characters.");
        else
            System.out.println("Do not Contain unique characters.");
        
     }
     static boolean checker(String input){
        boolean checker = true;
        int store = 0;
        
        for(int i = 0; i < input.length(); i++ ){
            
            if ((store & (1 << (input.charAt(i)-97))) > 0){
                System.out.println(store);
                return false;    
            }
            else
                store |= (1 << (input.charAt(i)-97));
        }
        return checker;
         
     }
}

File: 1.1.1.c
#include <stdio.h>
#include <string.h>

int main ()
{
  char input[100] = "abcdefh";
  char charList[256];
  int i = 0, flag = 0;
  int l = strlen (input);
  printf (" the value of l = %d ", l);
  while (i < 256)
    charList[i++] = 0;

  i = 0;

  while (i < l)
    {
      charList[input[i]]++;
      if (charList[input[i++]] > 1)
	{
	  flag = 1;
	  break;
	}
    }
  i = 0;
  while (i < 256)
    printf ("%4d", charList[i++]);
  if (flag == 0)
    printf ("\n The string contains unique characters only");
  else
    printf ("\n The string DOES NOT contain unique characters only");
  return 0;
}
